src/crystal/system/unix/login.cr

```diff
@@ -0,0 +1,11 @@
+require "c/unistd"
+
+module Crystal::System
+  def self.login
+    if LibC.getlogin
+      String.new(LibC.getlogin)
```

Comment:
Wouldn't it be better to store the result in a local variable instead of calling `LibC.getlogin` twice?